changelog-template.hbs

```diff
@@ -0,0 +1,17 @@
+
+{{#each releases}}
+
+  ## [{{title}}] {{date}}
+  {{#if summary}}
+    {{summary}}
+  {{/if}}
+  {{#each merges}}
+    - {{#if commit.breaking}}**Breaking change:** {{/if}}{{message}}{{#if href}} [`#{{id}}`]({{href}}){{/if}}
+  {{/each}}
+  {{#each fixes}}
+    - {{#if commit.breaking}}**Breaking change:** {{/if}}{{commit.subject}}{{#if href}} [`#{{id}}`]({{href}}){{/if}}
+  {{/each}}
+  {{#each commits}}
+    - {{#if breaking}}**Breaking change:** {{/if}}{{subject}}{{#if href}} [`#{{shorthash}}`]({{href}}){{/if}}
```

Comment:
These are from commits before moving this this repo which don't have a merge id.

for example, here is a sample output

```
## [v1.9.0] 2021-02-26
- Add build-css gulp task for testing [`#4`](https://github.com/hypothesis/frontend-shared/pull/4)
- Standup CI & publish action [`#2`](https://github.com/hypothesis/frontend-shared/pull/2)
- Remove karam.config.js from /lib output [`#3`](https://github.com/hypothesis/frontend-shared/pull/3)
- Add support for testing / lint / typecheck / format [`#1`](https://github.com/hypothesis/frontend-shared/pull/1)
- Modify github action to setup link to frontend-shared [`#f4c1cef`](https://github.com/hypothesis/frontend-shared/commit/f4c1cef38dd4d53a301fbf8ff7827ef709046d7b)
- Add svg-icon to frontend-shared package [`#0b02df5`](https://github.com/hypothesis/frontend-shared/commit/0b02df5f4b3f72d4348570dd05f8f962a3a1b92e)
- Add support for unit tests [`#bcbacdf`](https://github.com/hypothesis/frontend-shared/commit/bcbacdfcba564c3aaa848cbc6ab019f09c4b9bbf)
```